tests/contrib/kafka/test_kafka.py

```diff
@@ -16,6 +16,7 @@
 from ddtrace.contrib.kafka.patch import unpatch
 from ddtrace.filters import TraceFilter
 import ddtrace.internal.datastreams  # noqa: F401 - used as part of mock patching
+from ddtrace.internal.datastreams.kafka import PROPAGATION_KEY
```

Comment:
I'm personally not a big fan of using non-test constants in tests (it can lead to bugs - if for example  change the value for the constant, the tests won't fail).

Reply:
In this case, if the value changes, I don't want the test to fail.  Perhaps this is a call for another test to ensure that it doesn't change for cross-version compatability?